packages/nim/build.sh

```diff
@@ -40,7 +40,6 @@ termux_step_make() {
 	sed -i "s%\@LDFLAGS\@%${LDFLAGS}%g" config/nim.cfg
 	sed -i "s%\@CPPFLAGS\@%${CPPFLAGS}%g" config/nim.cfg
 
-	find -name "stdlib_osproc.nim.c" | xargs -n 1 sed -i 's',"/system/bin/sh\"\,\ 14","${TERMUX_PREFIX}/bin/sh\"\,\ 38",'g'
```

Comment:
Seems to just perform the same function as the [osproc.nim.patch](https://github.com/termux/termux-packages/blob/master/packages/nim/osproc.nim.patch) file and was causing problems since it operated on the intermediate c files (and the new update has different file names)